src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -178,6 +178,18 @@ TEST(BootstrapTest, GeneratedFilesMatch) {
   }
 }
 
+//test Generate in cpp_generator.cc
+TEST(BootstrapTest, OptionNotExist) {
+  cpp::CppGenerator generator;
+  DescriptorPool pool;
+  GeneratorContext *generator_context;
```

Comment:
One more thing I forgot to mention: could you set generator_context to nullptr? I don't think C++ is guaranteed to initialize it, so we have to explicitly set it to nullptr if we want it to be defined.

Reply:
OK ,thank you for reminding